src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -822,7 +824,8 @@ bool CommandRequestImpl::PrepareResultForMobileResponse(
           hmi_apis::Common_Result::WARNINGS,
           hmi_apis::Common_Result::WRONG_LANGUAGE,
           hmi_apis::Common_Result::RETRY,
-          hmi_apis::Common_Result::SAVED)) {
```

Comment:
@Ypostolov maybe better to provide function like 'bool IsResultOk(hmi_apis::Common_Result)' to remove duplicated logic?